### DISCUSSION TITLE
Welcome to OnlyM Discussions!

### DISCUSSION THREAD

**Discussion by AntonyCorbett** [AntonyCorbett/OnlyM#400] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. If you have a question or an idea, this is probably the place to start.

**Comment by JonGraef** [comment]
Hi Anthony,
I was just looking at presenting the Sunday talk from the JW Stream download file. I have set the start time, but wondered if you add an end time setting.
Jon Graef Bromsgrove Cong.

**Reply by AntonyCorbett** [reply]
@JonGraef - please create a new discussion.

**Comment by PaulAspinall** [comment]
Hi, 
I updated OnlyM and also downloaded the .net downloads. At the meeting this morning, I shared my screen to play the first song from JW Library and the sound was really distorted and I cannot fix it. 
It wouldn’t be linked to the .net downloads would it? 
I have uninstalled Zoom and reinstalled and it still isn’t fixed so I wanted to rule this out, as this was the only change I made between the midweek and Sunday meetings.

**Reply by AntonyCorbett** [reply]
@PaulAspinall - please create a new discussion.

**Comment by AntonyCorbett** [comment]
I am locking this post. Please create new discussions as needed: https://github.com/AntonyCorbett/OnlyM/discussions
